### ISSUE TITLE
fit to exponential, or draw exponential line to evaluate fit...

### ISSUE THREAD

**Issue by colinmegill** [colinmegill/ebola#7]

**Comment by metasoarous** [comment]
In the JS? All client side?

**Comment by colinmegill** [comment]
all client side yes

**Comment by trvrb** [comment]
And/or log the y-axis.  This will show exponential growth as a straight line and be easy to see visually:

![seir-curve](https://cloud.githubusercontent.com/assets/1176109/3952934/52901938-26e7-11e4-94c9-580125014655.png)

**Comment by metasoarous** [comment]
@trvrb - Hey! Thanks for the feedback! Please feel free to submit any other ideas you have as issues. Also, happy to give you repo access if you're interested in tinkering on this with us.

**Comment by trvrb** [comment]
That would be fun. I like tinkering with D3.  Would it be better for you to give me repo access or for me to fork and do pull requests?

**Comment by colinmegill** [comment]
@trvrb I added you to the repo to make it easy as we're moving really fast right now. The designers are taking a pass here, and we'll be at code complete Wed. night and deploy to Hacker News and other outlets Thursday morning of this week. Issues it is realistic to have done by then will be added to  Release, others to Enhancements. We'll add your name to the 'built by' on the site. Thanks for your help! The main purpose of this site is to be able to quickly assess the rate of change of infection on a daily basis, something I found lacking from the news reports that put out static graphs for that day only.
